src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONGTERMDISCOUNT = 50;
+  const LONGTERM = 7;
+  const SHORTTERMDISCOUNT = 20;
+  const SHORTTERM = 3;
```

Comment:
move these variables outside the function so that they are not recreated when the function is called